### ISSUE TITLE
Chunks not loading outside of Overworld with Render distance >120 chunks

### ISSUE THREAD

**Issue by doak-m** [Johni0702/bobby#222]
This has been happening to me for a week or two now, and I'm using the latest versions of all the mods I have on.
With a higher render distance (the max I went to was 256 chunks), everything seems to be working fine in the Overworld.
However, in the Nether and in the End, beyond ~120 chunks, every block turns pitch black.
Here's a video of me demonstrating (with mod list):

https://github.com/Johni0702/bobby/assets/105933226/4a6a9e44-1543-499f-8c29-c6ed40323309

**Comment by damster101** [comment]
Yeah, same issue for me. Exactly 120 chunks is the maximum render distance before rendering breaks in the Nether.
